auditlog/models/log.py

```diff
@@ -9,7 +10,11 @@ class AuditlogLog(models.Model):
     _order = "create_date desc"
 
     name = fields.Char("Resource Name", size=64)
-    model_id = fields.Many2one("ir.model", string="Model")
+    model_id = fields.Many2one(
+        "ir.model", string="Model", index=True, ondelete="set null"
+    )
+    model_name = fields.Char(string=u"Model Name", readonly=True)
+    model_model = fields.Char(string=u"Technical Model Name", readonly=True)
```

Comment:
@StefanRijnhart the `unicode` is probably not required here?

Reply:
Ah yes. As a matter of fact it wasn't necessary in Python 2.7, but they are particularly offensive in Python 3.x. Removed now.